industry/products.md

```diff
@@ -26,39 +26,62 @@ Raw models usually do not include the features included in many production syste
-| [Unbabel](/../industry/companies.md#unbabel) | Unbabel Translate | |
-| [Lilt](/../industry/companies.md#lilt) | Lilt Translate | |
-| [Tilde](/../industry/companies.md#tilde) | TildeMT | |
-| [Omniscien](/../industry/companies.md#omniscien-technologies) | Language Studio | |
-| [KantanMT](/../industry/companies.md#kantanmt) | Translate AI | |
-| [Pangeanic](/../industry/companies.md#pangeanic) | PangeaMT | |
-| TextShuttle | TextShuttle AI Translation | |
-| Globalese | Globalese | |
-| RWS Group | Language Weaver (Iconic) | &#10004; |
-| Mirai Translate | Mirai Translator | &#10004; |
-
+| Product | Self-serve |
+| --- | --- |
+| NiuTrans API by **[Xiaoniu Fanyi](/../industry/companies.md#xiaoniu-fanyi)** | ✔ |
+| [KantanMT](/../industry/companies.md#kantanmt) by **Keywords Studios** | ✔ |
+| **Google** Translate | ✔ |
+| **Microsoft** Translator Text API | ✔ |
+| **[LingvaNex](/../industry/companies.md#lingvanex)** Translator | ✔ |
+| **[Language I/O](/../industry/companies.md#language-io)** Translation API | ✔ |
+| **Yandex** Translate | ✔ |
+| **BytePlus** Translate | ✔ |
+| **Amazon** Translate | ✔ |
+| Xinyi by **[Zhongke Fanyu Technology](/../industry/companies.md#zhongke-fanyu-technology)** | ✔ |
+| **[AppTek](/../industry/companies.md#apptek)** API | |
+| [Language Weaver](/../industry/companies.md#language-weaver) [(Iconic)](/../industry/companies.md#iconic-translation-machines) by **RWS Group** | ✔ |
+| [ModernMT](/../industry/companies.md#modernmt) by **[Translated](/../industry/companies.md#translated)** | ✔ |
+| **IBM** Watson Language Translator | ✔ |
+| **[SYSTRAN](/../industry/companies.md#systran)** Translate Pro | ✔ |
+| Language Studio by **[Omniscien Technologies](/../industry/companies.md#omniscien-technologies)** | ✔ |
+| **[Lilt](/../industry/companies.md#lilt)** Translate | |
+| **[PROMT](/../industry/companies.md#promt)** Neural Translation Server | |
+| **[ImageTranslate](/../industry/companies.md#imagetranslate)** API | |
+| Kode AI by **Kodensha** | ✔ |
+| **Globalese** API | |
+| **[XL8](/../industry/companies.md#xl8)** Skroll | ✔ |
+| NICT NMT by **[Japan Patent Translation Co.](/../industry/companies.md#japan-patent-translation-co)** | |
+| **[Unbabel](/../industry/companies.md#unbabel)** Translate | ✔ |
+| **Baidu** Translate | ✔ |
+| **[Tilde](/../industry/companies.md#tilde)** MT | ✔ |
+| **[DeepL](/../industry/companies.md#deepl)** API | ✔ |
+| **Alibaba** Machine Translation | ✔ |
+| **[Lengoo](/../industry/companies.md#lengoo)** Translate | ✔ |
+| **Mirai** Translator | ✔ |
+| PangeaMT by **[Pangeanic](/../industry/companies.md#pangeanic)** | |
+| **[Lingua Custodia](/../industry/companies.md#lingua-custodia)** API | ✔ |
+| **[TextShuttle](/../industry/companies.md#textshuttle)** AI Translation | ✔ |
+| Lex Machina by **Hieronymus** | |
+| Judicio by **CrossLang** | |
+| **[Bering Lab](/../industry/companies.md#bering-lab)** MT | ✔ |
+| LETR by **Twigfarm** | |
 
 ## Machine translation aggregators
```

Comment:
"Machine translation" is understood here, same reason we don't use it for "Training and benchmarking tools"